src/Gui/NaviCube.cpp

```diff
@@ -354,6 +354,8 @@ NaviCubeImplementation::NaviCubeImplementation(Gui::View3DInventorViewer* viewer
     m_View3DInventorViewer = viewer;
     m_PickingFramebuffer = nullptr;
     m_Menu = createNaviCubeMenu();
+    flatButtonAnimation = {};
+    flatButtonTargetOrientation = {};
```

Comment:
These should be unnecessary, both of those objects have constructors that will run automatically.